enrollment.js

```diff
@@ -0,0 +1,26 @@
+window.easyEdu = {
```

Comment:
window.easyedu = window.easyedu || { ...

Reply:
also put all the code inside a lambda and auto call it, eg:

(function () {
  function validateParam() { ... }

  function enroll() { ... }

  window.easyedu = window.easyedu || function () { enroll.apply(this, arguments); };
})();